data/REP-11/voting-proposal.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+      "subspace": "bank",
+      "key": "DefaultSendEnabled",
+      "value": "false"
+    },
+    {
+      "subspace": "bank",
+      "key": "SendEnabled",
+      "value": "[{\"denom\":\"PLMNT\",\"enabled\":true},{\"denom\":\"STAKE\",\"enabled\":true}]"
```

Comment:
don't know if this is a problem but usually we lowercase the denominators: PLMNT -> plmnt and STAKE -> stake...